spec/browsers_json_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe 'browsers.json' do
+  before { system 'make browsers.json' }
```

Comment:
a few ideas:

only run once before the whole file
```suggestion
  before(:all) { system 'make browsers.json' }
```

only run if the file exist exist?
```
before { system 'make browsers.json' if !File.exists?(Rails.root.join('browsers.json')) }
```

Reply:
> a few ideas:

I'll go with the former, since the latter could cause false positives if the developer has a stale `browsers.json` that includes known keys (if rerunning would introduce unknown keys, for example).

Technically, the Make target is set up so that it should cache the result and not regenerate every time, but I think the test oughtn't assume that, and `before(:all)` seems like a reasonable expectation to prep all specs in the file.

Reply:
> a few ideas:

Switched to `before(:all)` in 0128ca461.